stdlib/Makefile

```diff
@@ -200,21 +197,23 @@ export AWK
 %.cmi: %.mli
 	$(CAMLC) $(COMPFLAGS) $(shell ./Compflags $@) -c $<
 
-stdlib__%.cmi: %.mli
-	$(CAMLC) $(COMPFLAGS) $(shell ./Compflags $@) -o $@ -c $<
+stdlib__%.cmi:
+	$(CAMLC) $(COMPFLAGS) $(shell ./Compflags $@) \
+	         -o $@ -c $(filter %.mli, $^)
```

Comment:
`$^` expands to all dependencies of the `.cmi`, and we are keeping only the `.mli` one(s?). Where does the dependency information come from here?

Reply:
It comes from `.depend` - sorry, I intended to post a second comment on the PR, but had to supervise a junior Zoom meeting 😆

Reply:
Could you add a comment in the source so that people reading the code and wondering about how these `.mli` dependencies know where to look to understand where they come from?

Reply:
Yes, of course!